python/cudf/cudf/core/index.py

```diff
@@ -128,6 +128,16 @@ def __init__(
         """
         pass
 
+    def drop_duplicates(self, keep="first", inplace=False, ignore_index=False):
+        """
+        Return Index with duplicate values removed
+        """
```

Comment:
Index `drop_duplicates` shouldn't have `inplace` or `ignore_index` kwargs. Also need to add `keep` kwarg to the docstring.